pkg/provider/kubernetes/k8s/event_handler.go

```diff
@@ -48,12 +48,10 @@ func objChanged(oldObj, newObj interface{}) bool {
 	}
 
 	if _, ok := oldObj.(*corev1.Endpoints); ok {
-		if endpointsChanged(oldObj.(*corev1.Endpoints), newObj.(*corev1.Endpoints)) {
-			return true
-		}
+		return endpointsChanged(oldObj.(*corev1.Endpoints), newObj.(*corev1.Endpoints))
 	}
 
-	return false
+	return true
```

Comment:
Should we keep this false, and invert the condition in line 46 instead?
```suggestion
	return false
```

Reply:
```
if oldObj.(metav1.Object).GetResourceVersion() != newObj.(metav1.Object).GetResourceVersion() {
	return true
}
```

Reply:
If we do that, it will return true as soon as the version will be different and the endpointsChanged will be useless.

Reply:
Ah yep.  Definitely overlooked that.